### PR TITLE
Charter changes: remove redundancies

### PR DESCRIPTION
**Pull request by MylesBorins**
This build on top of #384, which will need to land first. Please look at 
966ddef for the specific changes that are made in this PR.

The CommComm should reach consensus on this PR and then submit to the board for review.

### CODE REVIEW

**Comment by MylesBorins** [comment]
Note that this removes reference to the individual member director

/cc @williamkapke

File: Community-Committee-Charter.md
Comment (hackygolucky) [inline Community-Committee-Charter.md]: We should remove this requirement that an Individual Membership Director is automatically a member. @williamkapke is active in CommComm as a member regardless and I think that's a much more organic approach to input and information gathering.

**Comment by MylesBorins** [comment]
@hackygolucky and everyone else, please feel free to push changes on here.

The next board meeting is next week, so it might make sense to move forward getting approval of the changes in the other PR and work out consensus for additional changes here and attempt to get this in front of the board in the next meeting. Thoughts?

**Review by mhdawson — APPROVED** [review]
LGTM

**Review by amiller-gh — APPROVED** [review]
I'm happy with these as-is.

**Review by WaleedAshraf — APPROVED** [review]
LGTM after typo fix.

**Review by keywordnew — APPROVED** [review]
Nits aside, these changes LGTM as is 👍🏽

**Review by codeekage — APPROVED** [review]

**Comment by bnb** [comment]
>as represented on the Board by the existing Individual Membership Board Directors.

This part from line 9 is inaccurate. We need to update it 🤔

**Comment by bnb** [comment]
> Changes to CommComm membership should be posted in the agenda, and may be suggested as any other agenda item.

This part from line 23 hasn't historically been accurate – only additions to CommComm membership have been posted in the agenda. Not sure if this is something worth changing.

**Comment by bnb** [comment]
>The meeting shall be directed by the Individual Membership Directors.

This part from line 31 has historically never happened as far as I know. I suggest a modification:

>The meeting shall be directed by the Community Committee Chairperson.

**Comment by MylesBorins** [comment]
@bnb since the board meeting is in two days I would like to suggest we move forward with these changes exactly as laid out and then do another round of corrections to be landed in the next board meeting. Thoughts?

**Comment by bnb** [comment]
@MylesBorins I'm fine with that. Should I open a new PR?

**Comment by hackygolucky** [comment]
@bnb I know this was approved by the Board but I didn't realize we still had some questions from you. Are we good to remove the do-not-merge tag and merge this? I believe your questions are the blocker. From what I'm reading, you'd want to transfer comments from this into a new pull request as the next round of iteration.

**Comment by bnb** [comment]
@hackygolucky yup, you can remove do-not-merge if my questions are the blocker. If we merge this I can just spin up a new PR as soon as I have time :)

**Pull request merged**
